dashboard/config/initializers/markdown_handler.rb

```diff
@@ -6,8 +7,29 @@ module MarkdownHandler
     space_after_headers: true
   }
 
+  # Rewrite youtube iframe elements to use the fallback-player iframe instead
+  class YoutubeRewriter < Redcarpet::Render::HTML
+    def block_html(html)
+      doc = ::Nokogiri::HTML(html)
+      nodes = doc.xpath(%w(youtube youtubeeducation).map{|x| "//iframe[@src[contains(.,'#{x}.com/embed')]]"}.join(' | '))
+      nodes.each do |node|
+        if node['src']
+          regex = /(http:|https:)?\/\/(www\.)?(youtube|youtubeeducation)\.com\/embed\/([^"&?\/ ]{11})/
+          id = node['src'].match(regex)[4]
```

Comment:
Consider using named captures to make this more readable and robust (in case the regular expression changes)

http://ruby-doc.org/core-1.9.3/Regexp.html#method-i-named_captures